### PR TITLE
docs: Known issues

### PR DESCRIPTION
**Pull request by TansyEliot**
Fixes #[issue_no]

## Summary of Changes

Known issues fixes on README.md

## Additional context

Discord username (if different from GitHub): TansyEliot#9914

## Testing instructions

No testing necessary.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Review by pepperoni505 — CHANGES_REQUESTED** [review]
This should include the known issues on our Discord

File: README.md [hunk 1]
```diff
@@ -87,12 +87,9 @@ See [Contributing.md](.github/Contributing.md)
...
 ## Known Issues (Please note that most issues are being worked on and some of them may even be fixed in the master branch)
 
-- Captain's PFD may occasionally turn off during flight
+- Captain's PFD may occasionally turn off during flight.
 - No Smoking switch doesn't use a full range of motion.
-- F/CTL page does not have working speedbrake integration
-- BLEED page is not fully functional
-- Automatic ECAM page switching has minor bugs
-- APU/Engine fire covers cannot be retracted once opened
+- Automatic ECAM page switching has minor bugs.
```
Comment (pepperoni505) [inline README.md:92]:
```suggestion
- Automatic ECAM page switching has minor bugs.
- AP not following the flight plan (leaking input values affect, but don't disconnect the AP) [Workaround: Set dead-zones for your input device higher] 
- Autopilot goes direct to RWY on APP (same with the default A320) [Workaround: Use DIR to a waypoint or selected heading]
- Upper ECAM displays wrong THR levers position / N1 rating.
- Newly added ASOBO A320 liveries are incompatible with the A32NX mod. 
- Ruder keybindings not working (you have to set your keybinding to rudder axis right and left)
- Wing dips on landing (due to bad transition to direct law in flare, same with the default A320)  [Workaround use minimal aileron input on landing]
- Black screens / unable to start (conflict with another mod/livery or incorrect installation of the A32NX mod, use the [installer](https://api.flybywiresim.com/installer))
```

File: README.md [hunk 2]
```diff
@@ -87,12 +87,9 @@ See [Contributing.md](.github/Contributing.md)
 
 ## Known Issues (Please note that most issues are being worked on and some of them may even be fixed in the master branch)
 
-- Captain's PFD may occasionally turn off during flight
+- Captain's PFD may occasionally turn off during flight.
```
Comment (pepperoni505) [inline README.md:90]:
```suggestion
- Captain's PFD may occasionally turn off during flight

```

**Review by ImenesFBW — APPROVED** [review]
LGTM!

**Review by ZigTag — APPROVED** [review]
LGTM

**Pull request merged**